core/domain/value_generators_domain.py

```diff
@@ -67,13 +78,20 @@ def generate_value(self, *args, **kwargs):
 
 
 class Registry(object):
-    """Registry of all value generators."""
+    """Maintains a registry of all the value generators.
+
+    Attributes:
+        value_generators_dict: Dictionary mapping value generator class names
```

Comment:
Missing type info (see other docstrings for examples). I think you can do something like:

    value_generators_dict: dict(str : BaseValueGenerator). Dictionary mapping ...